server/vulnerabilities/nvd/cve.go

```diff
@@ -243,9 +253,15 @@ func checkCVEs(
 							}
 						}
 
+						resolvedVersion, err := getMatchingVersionEndExcluding(ctx, matches.CVE.ID(), softwareCPE.meta, dict, logger)
+						if err != nil {
+							level.Error(logger).Log(logKey, "error", "err", err)
```

Comment:
Do you foresee a lot of these being generated? If so, we may want to use `level.Debug`

Reply:
agreed, moved